### PR TITLE
CASMINST-6233 get CFS configs for specific management node types

### PR DESCRIPTION
**Pull request by leliasen-hpe**
# Description
Get CFS configs for specific management node types

# Checklist

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: workflows/iuf/operations/management-nodes-rollout/management-nodes-rollout.yaml [hunk 1]
```diff
@@ -98,23 +98,18 @@ spec:
...
-                    if [[ $? -ne 0 ]]; then
-                      exit 1 # could not find the desired cfs configuration
-                    else
-                      echo "NOTICE found CFS configuration:$configuration in CFS"
-                    fi
 
                     if $rebuild_workers; then
+                      # Check config exists
+                      config=$(echo $prepare_images_ouput | jq '.images | map(select(.configuration_group_names[] == "Management_Worker")) | .[].configuration' | tr -d '"')
+                      if [[ -z $config ]]; then echo "ERROR  no CFS configuration was recieved for 'Management_Worker' from prepare images stage"; exit 1; fi
```
Comment (JohnNavitskyHPE) [inline workflows/iuf/operations/management-nodes-rollout/management-nodes-rollout.yaml:105]: I'm guessing the error here would more accurately be that there was no image for management_worker?

File: workflows/iuf/operations/management-nodes-rollout/management-nodes-rollout.yaml [hunk 2]
```diff
@@ -98,23 +98,18 @@ spec:
...
+                      cray cfs configurations describe "$config" > /dev/null
+                      if [[ $? -ne 0 ]]; then
+                        exit 1 # could not find the desired cfs configuration
```
Comment (JohnNavitskyHPE) [inline workflows/iuf/operations/management-nodes-rollout/management-nodes-rollout.yaml:108]: It would be good to throw an error here.

**Review by JohnNavitskyHPE — APPROVED** [review]

File: workflows/iuf/operations/management-nodes-rollout/management-nodes-rollout.yaml (hunk 2 above)
Comment (leliasen-hpe) [reply workflows/iuf/operations/management-nodes-rollout/management-nodes-rollout.yaml:108]: The 'cray cfs configuration describe' provides a relatively good error if it is unable to find the configuration

Comment (rustydb) [reply workflows/iuf/operations/management-nodes-rollout/management-nodes-rollout.yaml:108]: Dennis merged this before we caught this feedback, apologies.

**Pull request merged**